### PR TITLE
Redesigned new encounter form  - take 2

### PR DESCRIPTION
**Pull request by zbig01**
Made changes and checked code to make sure that all previously committed
changes are included.

### CODE REVIEW

File: interface/forms/newpatient/common_help.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                <p><?php echo xlt("Clicking on the Add Issue button brings up the add issue pop-up box");?>.
+                <a href="#" class="css_button_small" style="float:none"><?php echo xlt("Add Issue");?></a>
+
+                <p><?php echo xlt("The default issue type is set to Problem");?>.
+
+                <p><?php echo xlt("For each issue type there is a list of commonly selected titles which are customizable for your clinic's specific needs");?>.
+
+                <p><?php echo xlt("If the desired title is not included, it can be typed into the text box below the list");?>.
+
+                <p><?php echo xlt("Select an appropriate ICD/CPT  code. This helps search for education material to give to the patient for that particular problem");?>.
```
Comment (bradymiller) [inline interface/forms/newpatient/common_help.php]: Remove double spacing in above constant and ensure none of the other constants in this script have double spacing

File: interface/forms/newpatient/common.php [hunk 2]
```diff
@@ -76,444 +76,501 @@ function sensitivity_compare($a, $b)
...
+    border-bottom-right-radius: 3px;;
+}
+.btn-group-pinch >.btn:nth-last-child(2):not(.dropdown-toggle) {
+    border-top-right-radius: 3px !Important;
+    border-bottom-right-radius: 3px !Important;
+}
+@media only screen and (max-width: 1024px) {
+    #visit-details [class*="col-"], #visit-issues [class*="col-"]{
+    width: 100%;
+    text-align:left!Important;
```
Comment (bradymiller) [inline interface/forms/newpatient/common.php]: Didn't we get rid of most of this embedded stuff already in prior stuff (like the forms)?

File: interface/forms/newpatient/common_help.php (hunk 1 above)
Comment (zbig01) [reply interface/forms/newpatient/common_help.php]: Done

File: interface/forms/newpatient/common.php (hunk 2 above)
Comment (zbig01) [reply interface/forms/newpatient/common.php]: @media rules are needed to keep page responsive, unfortunately due to the varying design of the pages the break point  differs for individual pages

Comment (bradymiller) [reply interface/forms/newpatient/common.php]: hi @zbig01 ,
I am referring to all the embedded styling above the @media section

Comment (zbig01) [reply interface/forms/newpatient/common.php]: I can get rid of them if I can add a few oe- classes to style_light.css and style_manila.css. Would you like me to do so.

Comment (bradymiller) [reply interface/forms/newpatient/common.php]: hi @zbig01 , but none of these look like oe- classes and instead override the way buttons are viewed/separated (which we decided in past to leave up to the primary styles). Let me know if I am wrong on this assumption, though.

Comment (bradymiller) [reply interface/forms/newpatient/common.php]: for example, what happened to the use of oe-opt-btn-group-pinch and oe-opt-btn-separate-left classes?

**Comment by robertdown** [comment]
@zbig01 theres been a discussion on the forums regarding where code should live (https://community.open-emr.org/t/documentation-and-codebase/10125). I think the documentation you’re writing has value, but not as an embedded help screen. Instead I’d like to see this kind of documentation on the wiki. I’m ok with the documentation staying in this PR since it’s already done, but would recommend not putting docs in the PR in the future and instead add them to the wiki. Note, the wiki is about to receive a *major* overhaul and should be much improved.

File: interface/forms/newpatient/common.php (hunk 2 above)
Comment (zbig01) [reply interface/forms/newpatient/common.php]: Sorry have been busy,will look into it and get back.

**Comment by zbig01** [comment]
I have redesigned the help pages and will create a new PR. Thanks

**Comment by bradymiller** [comment]
The new PR is specific to the help pages. Lets leave this up, since I still plan to review the code in interface/forms/newpatient/common.php
(and if @zbig01 does create another PR for this in future, then can close this)

**Comment by bradymiller** [comment]
@zbig01 ,
Just checking if it makes sense for me to rebase this to most recent codebase to make it simpler (ie. would remove all the styling file changes)?
-brady

**Comment by zbig01** [comment]
@bradymiller 
Please close this PR. I will submit a new PR for the New Encounter Form

**Pull request closed without merging**